lib/cucumber/multiline_argument/data_table.rb

```diff
@@ -75,7 +75,9 @@ def row(row)
         def eof; end
       end
 
-      NULL_CONVERSIONS = Hash.new(strict: false, proc: ->(cell_value) { cell_value }).freeze
+      # This is a Hash being initialized with a default value of a Hash, DO NOT REFORMAT TO REMOVE {}
+      # Future versions [3.4.0+] of ruby will interpret these as keywords and break.
+      NULL_CONVERSIONS = Hash.new({strict: false, proc: ->(cell_value) { cell_value }}).freeze
```

Comment:
Can you space out the arguments of the hash so `{ strict.....` and rebase with `main`. Rubocop "should" now pass